tech_soln/real_time_analytics.rst

```diff
@@ -0,0 +1,425 @@
+.. _introduction:
+
+Real Time Analytics
```

Comment:
Could we use the following for the opening?

Real Time Dashboards
#####################

Citus provides its users real-time responses over large datasets. One workload that
we commonly see at Citus involves powering real-time dashboards over events data.

For example, you could be a cloud services provider helping other businesses serve their
HTTP traffic. In this setup, every time one of your clients receives an HTTP request, you
receive a log record. You want to ingest all of those records and create an HTTP
analytics dashboard which gives your clients insights, such as the number HTTP errors
their site served. It's important that this data show up with as little latency as
possible so your clients can fix problems with their sites. It's also important to show
graphs of historical trends through a dashboard to your clients.